packages/venia-concept/src/components/ProductImageCarousel/thumbnail.js

```diff
@@ -10,19 +10,37 @@ class Thumbnail extends Component {
     static propTypes = {
         classes: PropTypes.shape({
             root: PropTypes.string
-        })
+        }),
+        isActive: PropTypes.bool,
+        item: PropTypes.shape({
+            label: PropTypes.string,
+            position: PropTypes.number,
+            disabled: PropTypes.bool,
+            file: PropTypes.string.isRequired
+        }),
+        onClickHandler: PropTypes.func
```

Comment:
Let's make this one `PropTypes.func.isRequired` since we call it directly in `onClickHandlerWrapper`.